tests/js/assets/js/yii.validation.test.js

```diff
@@ -0,0 +1,1598 @@
+describe('yii.validation', function () {
+    var VALIDATOR_SUCCESS_MESSAGE = 'should leave messages as is';
+    var VALIDATOR_ERROR_MESSAGE = 'should extend messages with according error(s)';
```

Comment:
Why is it unclear?

The validator adds a message to the messages array using [Array.prototype.push](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/push). The documentation for that method uses similar wording ("**adds** one or more elements to the end of an array"), and it's simpler and clearer than "extend", which makes me think of [jQuery.extend](https://api.jquery.com/jquery.extend/).

Secondly, "according error" sounds very unnatural and weird (it might not be grammatically incorrect, since according can be an adjective, but it's not the right word to use here). "Appropriate error" or "relevant error" or "correct error" is much better and clearer.